### PR TITLE
Feature/validitychecker

### PR DESCRIPTION
**Pull request by JulienChampagnol**

### CODE REVIEW

File: functions.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    List = []  # Initiaslizes an empty list
+    ObjectsList = GeodeObjects.ObjectsList() # Dict to loop through
+
+    for Object in ObjectsList.values():
+        values = Object['input']
+        for value in values:
+            list_creators = value.list_creators()
+            for creator in list_creators:  # Loop through
+                if creator not in List:  # If object's name isn't already in the list
+                    List.append(creator)  # Adds the object's name to the listlist
```
Comment (BotellaA) [inline functions.py:23]:
```suggestion
                    List.append(creator)  # Adds the object's name to the list
```

File: functions.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    List.sort()
```
Comment (BotellaA) [inline functions.py:24]: Do you really need the sorting?

Comment (BotellaA) [reply functions.py:24]: same question multiple times

File: functions.py (hunk 1 above)
Comment (BotellaA) [reply functions.py:23]: this typo is several times in the file. Do you really need all those comments?

File: functions.py (hunk 2 above)
Comment (JulienChampagnol) [reply functions.py:24]: Since OG doesn't sort the list itself, it allows the output file extensions to be sorted in the UI

**Pull request merged**